src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/NonEmptyAtclauseDescriptionCheck.java

```diff
@@ -95,13 +95,111 @@
  * class Test
  * {
  * &#47;**
- * * Violation for param "b". Tags "deprecated", "throws" are ignored.
+ * * Violation for param "b".
  * * &#64;param a Some javadoc // OK
  * * &#64;param b
- * * &#64;deprecated
+ * *&#47;
+ * public int method(String a, int b)
+ * {
+ * return 1;
+ * }
+ * }
+ * </pre>
+ * <p>
+ * To configure the check to validate {@code @return} tags:
+ * </p>
+ * <pre>
+ * &lt;module name="NonEmptyAtclauseDescription"&gt;
+ *   &lt;property name="javadocTokens" value="RETURN_LITERAL"/&gt;
+ *  &lt;/module&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * class Test
+ * {
+ * &#47;**
+ * * Violation for return tag.
+ * * &#64;param a Some javadoc
+ * * &#64;return
+ * *&#47;
+ * public int method(String a)
+ * {
+ * return 1;
+ * }
+ * }
+ * </pre>
+ * <p>
+ * To configure the check to validate {@code @throws} tags:
+ * </p>
+ * <pre>
+ * &lt;module name="NonEmptyAtclauseDescription"&gt;
+ *    &lt;property name="javadocTokens" value="THROWS_LITERAL"/&gt;
+ *   &lt;/module&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * class Test
+ * {
+ * &#47;**
+ * * Violation for throws tag.
```

Comment:
lets make javadoc content to be same in all examples.
please use single line comments to mark violation

```
class Test
{
  // violation 4 lines below on 'return' tag.
  /**
  * Some summary.
  *  @param a Some javadoc
  * @return
```

Reply:
@romani I did'nt understand what are you suggesting. Are you saying just to use single line comment to mark voilation in each example.
like replace this  ``` * Violation for return tag. ```
with this ``` //violation ```

Reply:
Yes, attention to `4 lines belo`

Reply:
Ok, on it